ansible-playbooks/roles/connect-aks-to-acm-hub/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+---
+
+- name: Create private endpoint
+  command: |
+    az network vnet list --subscription {{ AKS_SUB }} -g {{ AKS_MRG }} --query [].name -o tsv --only-show-errors
+  register: aksvnet_out
+
+- name: set fact
+  set_fact:
+    AKS_VNET: '{{ aksvnet_out.stdout }}'
+
+- name: Get AKS subnet
+  shell: |-
+    az network vnet subnet list --resource-group {{ AKS_MRG }}  --subscription {{ AKS_SUB }} --vnet-name {{ AKS_VNET }} --query [].name -o tsv --only-show-errors | grep 'aks'
+  register: akssubnet_out
+
+- name: set fact
+  set_fact:
+    AKS_SUBNET: '{{ akssubnet_out.stdout }}'
+
+- name: Get HUB DNS Zone
+  command: |-
+    az network private-dns zone list --subscription {{ HUB_SUB }} -g {{ HUB_RG }} --query [].name -o tsv --only-show-errors
```

Comment:
OK on this but we have to keep on mind that this works 'cause in the HUB there's only one private-dns zone.

Reply:
yep